x-pack/plugins/gis/public/angular/get_initial_query.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const DEFAULT_QUERY_LANGUAGE = 'kuery';
```

Comment:
Pretty sure this is going to have to come from `config.get('search:queryLanguage')`

Reply:
We wanted to default all users to kuery regardless of advanced settings. The maps application is a new app and we did not want to start users with lucene. This is intentional.

Reply:
We are switching the default to KQL in an open PR. I would think that it would be best to not ignore the advanced setting here in case there are certain users who want to change the default back to Lucene, or if we end up adding additional query languages (SQL, etc.) in the future. But that's just my $0.02.

Reply:
Good to know. I will pull from the config then